test/e2e/destroypod_test.go

```diff
@@ -43,56 +44,64 @@ import (
 )
 
 const (
-	timeoutExpectedOutput  = "Slept for 0 milliseconds"
-	revisionTimeoutSeconds = 45
-	timeoutRequestDuration = 35 * time.Second
+	timeoutExpectedOutput   = "Slept for 0 milliseconds"
+	revisionTimeoutSeconds  = 45
+	timeoutRequestDuration  = 35 * time.Second
+	activatorRespawnTimeout = 5 * time.Minute
 )
 
-func TestDestroyPodInflight(t *testing.T) {
-	t.Parallel()
-	cancel := logstream.Start(t)
-	defer cancel()
-
-	clients := Setup(t)
+// testToDestroy for table-driven testing.
+var testToDestroy = []struct {
+	name   string
+	rmFunc func(*test.Clients) error
+}{
+	// Destroy pod which is receiving the requests.
```

Comment:
I think the names of the func's are descriptive enough to make comments redundant.